client/python/lib/test/test_check_tls_config.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+
+from ovmsclient.tfs_compat.grpc.serving_client import  _check_tls_config
+from config import TLS_CONFIG_VALID, TLS_CONFIG_INVALID, PATH_VALID
+
+@pytest.mark.parametrize("tls_config, isfile_called_count", TLS_CONFIG_VALID)
+def test_check_address_valid(mocker, tls_config, isfile_called_count):
+    mock_method = mocker.patch('os.path.isfile', side_effect=is_valid_path)
+    _check_tls_config(tls_config)
+    assert mock_method.call_count ==  isfile_called_count
+
+@pytest.mark.parametrize("tls_config, expected_exception, expected_message, isfile_called_count", TLS_CONFIG_INVALID)
+def test_check_address_invalid(mocker, tls_config, expected_exception, expected_message, isfile_called_count):
+    mock_method = mocker.patch('os.path.isfile', side_effect=is_valid_path)
+    with pytest.raises(expected_exception) as e_info:
+        _check_tls_config(tls_config)
+        assert str(e_info.value) == expected_message
+    
+    assert mock_method.call_count ==  isfile_called_count
+
+def is_valid_path(path):
+    if path == PATH_VALID:
+        return True
+    return False
```

Comment:
I just though that we could drop this approach and remove `PATH_VALID` value. 
Negative cases could have bool value like: `is_path_valid`  and in mocker side effects you could go with:
`mocker.patch("path/to/file", side_effect=lambda path : is_path_valid)`